awd/deposit.py

```diff
@@ -94,19 +101,36 @@ def deposit(
     date = datetime.today().strftime("%m-%d-%Y %H:%M:%S")
     s3_client = s3.S3()
     sqs_client = sqs.SQS()
+    dynamodb_client = dynamodb.DynamoDB()
```

Comment:
This may cause some confusion. There's the instance variable dynamodb and then there's also dynamodb_client.
Was there a reason functions `doi_to_be_added' and `doi_to_be_retired' were not left in deposit.py?

Reply:
Good point, I'll clean this up